corehq/apps/app_manager/templates/app_manager/partials/build_profiles_select.html

```diff
@@ -1,8 +1,8 @@
 {% load i18n %}
 <div class="row">
   <div class="col-sm-3">
-    {% trans 'Build Profile' %}:
-    <select id="build-profile-select-for-multimedia" class="form-control">
+    <b>{% trans 'Build Profile' %}:</b>
```

Comment:
Bold tags aren't semantic. The way to do this UI is `form-inline`, see docs here: https://bootstrapdocs.com/v3.0.3/docs/css/#forms-inline

Reply:
https://github.com/dimagi/commcare-hq/pull/26032/commits/91ae2fc17a7afcb3776d48fee16d192df21a734c